crates/nargo_cli/src/cli/prove_cmd.rs

```diff
@@ -35,6 +35,10 @@ pub(crate) struct ProveCommand {
     /// The name of the circuit build files (ACIR, proving and verification keys)
     circuit_name: Option<String>,
 
+    /// The name of the toml file which contains the inputs for the prover
+    #[clap(long, short, default_value = PROVER_INPUT_FILE)]
+    prover_name: Option<String>,
```

Comment:
Can you include this on all the commands which require this file? Off the top of my head this should just be `execute`.

We should also add a similar argument for `Verifier.toml`.

Reply:
>We should also add a similar argument for `Verifier.toml`.

In this PR or another one?

Reply:
Agreed